README.md

```diff
@@ -85,6 +85,8 @@ end
 ```julia
 julia> using PortAudio, SampledSignals, LibSndFile
 
+julia> using FileIO: load, save, loadstreaming, savestreaming
```

Comment:
```suggestion
```

Having a separate streaming example would be nice eventually, but the first priority should be to make the example in the README actually run properly!